### PR TITLE
add needed jvm opts to run cassandra tests under java11+; see #598

### PR DESCRIPTION
**Pull request by hennr**
Hi @bennetelli , togglz does build properly with this on my machine.
If you'd change fail-fast: true to false I think you should see the java 8 and 11 builds pass in a matrix build.

### CODE REVIEW

File: benchmarks/pom.xml [hunk 1]
```diff
@@ -51,6 +51,8 @@
             <artifactId>curator-test</artifactId>
             <version>5.2.0</version>
         </dependency>
+        <!--        brauchen wir hier hier nur die api? nur -simple? nur logback?  nur provided? -->
+        <!--        https://repo1.maven.org/maven2/org/slf4j/ -->
```
Comment (bennetelli) [inline benchmarks/pom.xml:55]: good question. I don't know. You can give it a try if you want :)

**Review by bennetelli — APPROVED** [review]

**Comment by bennetelli** [comment]
Many thanks for your PR :) I always wanted to upgrade the cassandra dependencies to achieve this. Good that's also possible without upgrading. Nice work!:)

File: benchmarks/pom.xml (hunk 1 above)
Comment (hennr) [reply benchmarks/pom.xml:55]: Whoops, these logging changes were not planned to be in this PR as well ;)

Sorry for that, I guess I have to dig into this as my next todo then :)

**Comment by bennetelli** [comment]
Works with java 8 and 11 on my machine.

**Comment by bennetelli** [comment]
@hennr now I (or we :-) ) need to replace the jdk internal nashorn engine to be able to build jdk 13+

**Comment by hennr** [comment]
> @hennr now I (or we :-) ) need to replace the jdk internal nashorn engine to be able to build jdk 13+

Yeah, well. Let's see 😏

**Pull request merged**